packages/api-derive/src/balances/fees.ts

```diff
@@ -29,19 +27,25 @@ const ZERO = new BN(0);
 export function fees (api: ApiInterface$Rx) {
   return (): Observable<DerivedFees> => {
     return (
-      // HACK This is a very quick fix to actually get the types in to continue with
-      // other work - so as of now, the fees are completely broken
-      api.query.balances.creationFee
-        ? api.queryMulti([
+      api.consts.balances
```

Comment:
I think this will throw on older nodes `api.consts is undefined`

Reply:
or not, actually

Reply:
I tested them with matching nodes for the conditions, so I hope they don't. :)
But you're right.... they should. Even looking at this gives me a "Cannot get property x of undefined"-error.